lib/collection.js

```diff
@@ -724,7 +714,7 @@ define.classMethod('bulkWrite', { callback: true, promise: true });
  * @typedef {Object} Collection~insertWriteOpResult
  * @property {Number} insertedCount The total amount of documents inserted.
  * @property {object[]} ops All the documents inserted using insertOne/insertMany/replaceOne. Documents contain the _id field if forceServerObjectId == false for insertOne/insertMany
- * @property {ObjectId[]} insertedIds All the generated _id's for the inserted documents.
+ * @property {ObjectId[]} insertedIds Map of the index of the inserted document to the id of the inserted document.
```

Comment:
We should change the jsdoc type to `{Object.<string, ObjectId}`

Reply:
agreed, but I think its actually `{Object.<Number, ObjectId>}`

Reply:
Right, that :)

Reply:
👍